ops-iam/20220929.md

```diff
@@ -0,0 +1,31 @@
+# 2022-09-29
+## Participants - use of github handle is preferred
+* @fkr
+* @matfechner
+* @itrich
+* @lossack
+* @juanptm
+* @o-otte
+* @master-caster (bzapiec)
+* @jnull
+* @joshmue
+* @garloff
+## Recurring items
+* Who is presenting in weekly community call and transforming etherpad to minutes in GitHub?
+   @joshmue
+* Anything else to add to the agenda?
+  * IAM Workshop
+## Agenda
+
+### Moving from R3 to R4 cycle
+https://github.com/SovereignCloudStack/issues/issues?q=is%3Aopen+is%3Aissue+label%3AOps+milestone%3A%22R3+%28v4.0.0%29%22
+=> OVN Issues marked as R3, discussion on wether to move them to R4 or R5.
+
+@garloff raises the point, that in the past, this was something vital for the Operators so that if they need it, we should see that we deliver this as R4 and not just move it to R5.
+We should talk to PCO, since they are heavy OVN user.
+
+### machine_identity (@joshmue)
+https://github.com/SovereignCloudStack/Docs/pull/140
```

Comment:
Add-in: We agreed that we want some kind of ADR process and establish and use it during the R4 cycle. We had a discussion on how ADRs would be reviewed and decided upon. First ADR should actually document what the (lazy) consensus and decision process is. @horazont volunteerd to make a proposal.

Reply:
thanks for the addition @garloff

Reply:
https://github.com/SovereignCloudStack/Docs/pull/143

Reply:
Incorperated that into the minutes.